ufo2fontir/src/toir.rs

```diff
@@ -350,8 +387,53 @@ mod tests {
         };
         // Switchy switchy!
         assert_eq!(
-            to_ir_component(&c).transform,
+            to_ir_component(&c, None).transform,
             Affine::new([0.4366, -0.4366, 0.4415, 0.4425, 282.0, 5.0])
         );
     }
+
+    /// Test parsing component anchors from glyphsLib's ComponentInfo in glyph lib.
+    ///
+    /// This tests that UFO files exported by glyphsLib with explicit component
+    /// anchors (e.g., for ligature attachment) are correctly parsed.
+    #[test]
+    fn component_anchors_from_lib() {
+        use plist::Value;
+
+        // Create a glyph with ComponentInfo in its lib
+        let mut glyph = norad::Glyph::new("test");
+
+        // Simulate glyphsLib's ComponentInfo structure:
+        // [{"name": "aa", "index": 0}, {"name": "acutecomb", "index": 1, "anchor": "top_2"}]
+        // See: <https://github.com/googlefonts/glyphsLib/blob/de5b4e34/Lib/glyphsLib/builder/components.py#L85-L93>
+        let component_info = Value::Array(vec![
+            Value::Dictionary(
+                [
+                    ("name".to_string(), Value::String("aa".to_string())),
+                    ("index".to_string(), Value::Integer(0.into())),
+                ]
+                .into_iter()
+                .collect(),
+            ),
+            Value::Dictionary(
+                [
+                    ("name".to_string(), Value::String("acutecomb".to_string())),
+                    ("index".to_string(), Value::Integer(1.into())),
+                    ("anchor".to_string(), Value::String("top_2".to_string())),
+                ]
+                .into_iter()
+                .collect(),
+            ),
+        ]);
```

Comment:
might be easier to just inline a PLIST and deserialize it? But not worth rewriting..